packages/core/src/types.ts

```diff
@@ -1,5 +1,52 @@
 import { ImageAd, Spoc, UaTile } from "@heyapi"
 
+declare global {
+  var __gpp: GPPFunction | undefined
+}
+
+export interface GPPCommand {
```

Comment:
Might be nice to link here to the IAB GPP docs, as we do below for the IAB standard formats...